proto/exocore/operator/v1/validator.proto

```
@@ -0,0 +1,90 @@
+syntax = "proto3";
+package exocore.operator.v1;
+
+import "gogoproto/gogo.proto";
```

buf

import "gogoproto/gogo.proto": file does not exist

(COMPILE)